tests/queries/0_stateless/02961_output_format_compress_params.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO FUNCTION file('test_02961.csv', 'CSV', 'x UInt64', 'zstd') SELECT number FROM numbers(1000000) SETTINGS output_format_compression_level = 10, output_format_compression_zstd_window_log = 30, engine_file_truncate_on_insert = 1;
```

Comment:
Can you add a random prefix to the file so the `Stateless tests flaky check` will work correctly?

Reply:
I added `engine_file_truncate_on_insert = 1` but I guess stateless tests flaky check also randomise settings.